src/views/pages/polygon/honestwork/index.ejs

```diff
@@ -26,6 +33,7 @@ the initial 🚜 APR's will be low and will be increased in a logaritmic model b
 
 Thank y'all in advance it ain't much but you know it!!
 this ain't no financial advise consider this as a Meme farmable Token please Ape only what your degen minds consider SAFU <3
+One Love to y'all <3
```

Comment:
you can't use the plain < character in html

Reply:
thank you!!!!!